src/client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.8.0"
+version = "0.8.1"
```

Comment:
why 0.8.1 ?

Reply:
I must have done that unconsciously. I will need to yank that crate